domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/RepositoryInterface.kt

```diff
@@ -1,6 +1,6 @@
-package com.bottlerocketstudios.brarchitecture.data.repository
+package com.bottlerocketstudios.brarchitecture.domain.models
 
 /**
  * [Marker interface](https://en.wikipedia.org/wiki/Marker_interface_pattern) for all [Repositories](Repository) used in the app to quickly/easily find them.
  */
-interface Repository
+interface RepositoryInterface
```

Comment:
@VanyaHuamanBottleRocket I see why you refactored this but would prefer we keep this interface just named `Repository` and rename the other `Repository` data class to `GitRepository` (or similar). The reason is that new projects will end up deleting the project specific code (the git repository data class) and keep the Repository interface marker class.

Reply:
Renaming